test/micro/org/openjdk/bench/jdk/incubator/foreign/points/support/PanamaPoint.java

```diff
@@ -51,14 +51,15 @@
 
     static {
         CLinker abi = CLinker.getInstance();
-        LibraryLookup lookup = LibraryLookup.ofLibrary("Point");
+        System.loadLibrary("Point");
+        SymbolLookup lookup = SymbolLookup.loaderLookup();
         MH_distance = abi.downcallHandle(
             lookup.lookup("distance").get(),
             methodType(double.class, MemorySegment.class, MemorySegment.class),
             FunctionDescriptor.of(C_DOUBLE, LAYOUT, LAYOUT)
         );
         MH_distance_ptrs = abi.downcallHandle(
-            lookup.lookup("distance_ptrs").get(),
+                lookup.lookup("distance_ptrs").get(),
```

Comment:
Spurious white space
```suggestion
            lookup.lookup("distance_ptrs").get(),
```